docs/diagrams/UiClassDiagram.puml

```diff
@@ -33,25 +35,31 @@ UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
 MainWindow *-down-> "1" ResultDisplay
 MainWindow *-down-> "1" PersonListPanel
+MainWindow *-down-> "1" AppointmentListPanel
 MainWindow *-down-> "1" StatusBarFooter
 MainWindow --> "0..1" HelpWindow
 
 PersonListPanel -down-> "*" PersonCard
+AppointmentListPanel -down-> "*" AppointmentCard
 
 MainWindow -left-|> UiPart
 
 ResultDisplay --|> UiPart
 CommandBox --|> UiPart
 PersonListPanel --|> UiPart
 PersonCard --|> UiPart
+AppointmentListPanel --|> UiPart
+AppointmentCard --|> UiPart
 StatusBarFooter --|> UiPart
 HelpWindow --|> UiPart
 
 PersonCard ..> Model
```

Comment:
Would it be better to replace line 50  with `PersonCard -down--|> UiPart`, so it looks neater?

Currently `PersonCard --|> UiPart` looks like this:
<img width="632" alt="Screenshot 2024-10-22 at 23 17 22" src="https://github.com/user-attachments/assets/56a9895a-eefd-43b6-a984-8c7c89c7828c">

Changing to `PersonCard -down--|> UiPart` would look like this:
<img width="590" alt="Screenshot 2024-10-22 at 23 17 51" src="https://github.com/user-attachments/assets/7904b310-283a-461e-96e2-a3bb4c96a584">

Reply:
Looks a lot nicer! I'll make the changes now.